src/Pages/Cart/Cart.js

```diff
@@ -0,0 +1,41 @@
+import { useData } from "../../Contexts";
+import { CartCard } from "../../Components";
+import "./Cart.css";
+
+export function Cart() {
+  const { itemsInCart } = useData();
+  let total = 0;
+  return (
+    <>
+      {itemsInCart.length === 0 ? (
+        <div>
+          <h1>Cart is Empty</h1>
+        </div>
+      ) : (
+        <div className="App" style={{ display: "flex", flexWrap: "wrap" }}>
+          {itemsInCart.map((item) => (
+            <CartCard product={item} />
```

Comment:
you can provide `key` here only, checkout this link : [https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys)